index.html

```diff
@@ -233,17 +236,19 @@ <h2>Languages</h2>
                 
                 <li class="framework-skill" id="framework-container"> 
                     <h3 class="framework-skill-header">Frameworks</h3>
-                    <img src="./Images/right.png" alt="">
+                    <img src="./Images/right.png" alt="" class="right-chevron">
                 </li>
 
                 <img src="Images/Separator Bottom.png" alt="" id="separator-img">
 
                 <li class="framework-skill">
 
                     <h3 class="framework-skill-header">Skills</h3>
-                    <img src="./Images/right.png" alt="">
+                    <img src="./Images/right.png" alt="" class="right-chevron">
                 </li>
 
+                <img src="Images/Separator Bottom.png" alt="" id="separator-img2">
+
             </ul>
 
         </section>
```

Comment:
### I write the review 😃 for this part here as I can not write the code there 😊
- [ ] You did a great job 👌on the project, but I see that the background image of the footer is not placed like the Figma Design as you see here the background image is hidden on the page, so please 🚀😃 try to use the background-position or any property to place it in the correct position  😊💯
- [ ] Also consider 😃 fixing the place of the background image in the footer at the mobile version as you see it is hidden on the page and not correctly placed like the Figma, so let's try to place it like the Figma to make it nice 👌😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

> In the desktop version 🚀💯 

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/235677716-9db5f065-8a63-48ee-a392-0075a0dd8b82.png)  |     ![image](https://user-images.githubusercontent.com/83360636/235677611-67596e1e-6bd0-4965-94aa-57aa9cecd770.png)   |

> In the mobile version 💯👌 

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/235681458-1b60f36b-bcb9-4270-a87b-6588e63c96ba.png) |      ![image](https://user-images.githubusercontent.com/83360636/235681593-c698f819-6192-4527-8c9c-4040e419fc14.png)   |